recipes/onetbb/all/conandata.yml

```diff
@@ -5,3 +5,14 @@ sources:
   "2021.3.0":
     url: "https://github.com/oneapi-src/oneTBB/archive/v2021.3.0.tar.gz"
     sha256: "8f616561603695bbb83871875d2c6051ea28f8187dbe59299961369904d1d49e"
+patches:
+  "2021.6.0":
+    - patch_description: "cherry-pick upstream patch: avoid android compile errors for current NDKs on releases <= 2021.6.0"
+      patch_type: "backport"
+      patch_source: "https://github.com/oneapi-src/oneTBB/pull/716.diff"
```

Comment:
I love links! https://github.com/oneapi-src/oneTBB/pull/716 thank you!

Reply:
You are welcome :)